back/db/migrate/20220531123916_create_home_pages.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateHomePages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :home_pages, id: :uuid do |t|
+      t.boolean :header_avatars_enabled, default: true, null: false
```

Comment:
Is it fka `display_header_avatars`? On the design, we still use `Display avatars`?

Reply:
Formerly known as :) 

> Do you suggest changing the name of the column?

Not really suggesting, just wondering why you renamed `display_header_avatars` to `header_avatars_enabled`.
Perfectly fine as it is now :)